src/Icons/doc/index.rst

```diff
@@ -472,6 +472,20 @@ In production, you can pre-warm the cache by running the following command:
 This command looks in all your Twig templates for ``ux_icon()`` calls and
 ``<twig:ux:icon>`` tags and caches the icons it finds.
 
+Add ``-v`` to the command to see a list of icons being cached and potential
+missing icons:
+
+.. code-block:: terminal
+
+    $ php bin/console ux:icons:warm-cache -v
+
+.. caution::
+
+    The process to find icons to cache in your Twig templates is imperfect. It
```

Comment:
Maybe advise Lock command here? It shows details (-vv) on wether the IconSet does not exist (99% of the false positive extracted from templates) or if there is a real problem with an icon.